doc/configuration.md

```diff
@@ -19,7 +19,7 @@ Parameters controlling the Cluster-API management server (capi management server
 |--------------------------|-----------------|------------|-------------------|-------------------------------------------------------------------------------|
 | `prefix`                 |                 | SCS        | `capi`            | Prefix used for OpenStack resources for the capi mgmt node                    |
 | `kind_flavor`            |                 | SCS        | `SCS-2V-4`        | Flavor to be used for the k8s capi mgmt server                                |
-| `image`                  |                 | SCS        | `Ubuntu 22.04`    | Image to be deployed for the capi mgmt server                                 |
+| `image`                  |                 | SCS        | `Ubuntu 22.04`    | Image for the capi mgmt server. Use `Ubuntu 22.04` or `Debian 12`             |
```

Comment:
Do we support Ubuntu 20.04 and Debian 11? I can see `image = Ubuntu 20.04` in some environments

Reply:
Thanks for the explanation, I am fine with that